apostrophecms/apostrophe
# Rescale Task Bugfix

## Summary

Calling the task `@apostrophecms/attachment:rescale` was returning multiple errors. One was `Primise.promisify is not a function`. This fix aims to have the task working.

Get `Promise` from `bluebird` as is used further below.
Catch missing `file.crops` that would else throw an error.


## What are the specific steps to test this change?

*For example:*
1. Have some images. 
2. Run the task `@apostrophecms/attachment:rescale`
3. It should rescale the images and not throw errors everywhere
4. Check that all images have been rescaled properly

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:
